lib/generators/avo/install_generator.rb

```diff
@@ -15,6 +15,10 @@ def create_initializer_file
         template "initializer/avo.tt", "config/initializers/avo.rb"
         directory File.join(__dir__, "templates", "locales"), "config/locales"
       end
+
+      def create_user_resource
+        unless "User".safe_constantize.nil? run "./bin/rails generate avo:resource user"
```

Comment:
This is not bad, but we have a way to invoke generators programmatically.

So instead of `run ....` we could use `Rails::Generators.invoke("avo:resource", ["user", "-q"], {destination_root: Rails.root})`.

Reply:
Also, instead of `"User".safe_constantize.nil?` which might raise some errors in some scenarios we could use `defined?("User").present?`.